src/dbcp/data_mart/__init__.py

```diff
@@ -64,8 +66,8 @@ def create_data_marts(args):  # noqa: max-complexity=11
     with engine.connect() as con:
         for table in metadata.sorted_tables:
             logger.info(f"Load {table.name} to postgres.")
-            df = enforce_dtypes(data_marts[table.name], table.name, "data_mart")
-            df = dbcp.helpers.trim_columns_length(df)
+            df = dbcp.helpers.trim_columns_length(data_marts[table.name])
+            df = enforce_dtypes(df, table.name, "data_mart")
```

Comment:
We should trim the column names before we apply dtypes because we pull the type information from the sqlalchemy tables which use the postgres column names.